src/openrct2/world/SpriteBase.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "../common.h"
+#include "Location.hpp"
```

Comment:
Move this to the cpp and just forward declare `CoordsXYZ` here.
```suggestion

class CoordsXYZ;
```

Reply:
What is the advantage of that?

Reply:
You move the include to the source file and if you ever change `Location.hpp` content, you won't trigger recompilation of everyone that includes `SpriteBase`.